modules/highgui/src/window_framebuffer.hpp

```diff
@@ -0,0 +1,132 @@
+#ifndef OPENCV_HIGHGUI_WINDOWS_FRAMEBUFFER_HPP
+#define OPENCV_HIGHGUI_WINDOWS_FRAMEBUFFER_HPP
+
+#include "precomp.hpp"
+#include "backend.hpp"
+
+#include <linux/fb.h>
+#include <linux/input.h>
+
+#include <termios.h>
+
+namespace cv { namespace highgui_backend {
+
+enum OpenCVFBMode{
+    FB_MODE_EMU,
+    FB_MODE_FB,
+    FB_MODE_XVFB
+};
+
+class FramebufferBackend;
+class CV_EXPORTS FramebufferWindow : public UIWindow
+{
+    FramebufferBackend &backend;
+    std::string FB_ID;
+    Rect windowRect;
+
+    int flags;
+    Mat currentImg;
+
+public:
+    FramebufferWindow(FramebufferBackend &backend, int flags);
+    virtual ~FramebufferWindow();
+
+    virtual void imshow(InputArray image) override;
+
+    virtual double getProperty(int prop) const override;
+    virtual bool setProperty(int prop, double value) override;
+
+    virtual void resize(int width, int height) override;
+    virtual void move(int x, int y) override;
+
+    virtual Rect getImageRect() const override;
+
+    virtual void setTitle(const std::string& title) override;
+
+    virtual void setMouseCallback(MouseCallback onMouse, void* userdata /*= 0*/) override;
+
+    virtual std::shared_ptr<UITrackbar> createTrackbar(
+            const std::string& name,
+            int count,
+            TrackbarCallback onChange /*= 0*/,
+            void* userdata /*= 0*/
+    ) override;
+
+    virtual std::shared_ptr<UITrackbar> findTrackbar(const std::string& name) override;
+
+    virtual const std::string& getID() const override;
+
+    virtual bool isActive() const override;
+
+    virtual void destroy() override;
+}; // FramebufferWindow
+
+class CV_EXPORTS FramebufferBackend: public UIBackend
```

Comment:
`getName` from the base class is not implemented.
https://github.com/opencv/opencv/blob/d8f0838fa35b879e24196feafbf9dd1d683f0af0/modules/highgui/src/backend.hpp#L109

Reply:
the method appeared after the fork
I synchronized my fork and added a method implementation